spec/features/user/user_can_authenticate_with_github_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe "As a logged in user " , type: :feature do 
+
+  describe "When I connect through github " do 
+
+    before(:each) do 
+      OmniAuth.config.test_mode = true
```

Comment:
I like this line. I didn't recall seeing this in the lesson - I'll have to go back and check it!